crates/blockifier/src/test_utils/struct_impls.rs

```diff
@@ -250,13 +250,8 @@ impl NativeContractClassV1 {
         // memory space.
         fn compile_and_load(
             sierra_program: &cairo_lang_sierra::program::Program,
-        ) -> Result<AotNativeExecutor, cairo_native::error::Error> {
-            let native_context = cairo_native::context::NativeContext::new();
-            let native_program = native_context.compile(sierra_program)?;
-            Ok(AotNativeExecutor::from_native_module(
-                native_program,
-                cairo_native::OptLevel::Default,
-            ))
+        ) -> Result<ContractExecutor, cairo_native::error::Error> {
+            ContractExecutor::new(sierra_program, cairo_native::OptLevel::Default)
```

Comment:
Inline `compile_and_load`

Reply:
Removed `compile_and_load` and just used `ContractExecutor::new`.